mp/src/game/server/cbase.cpp

```diff
@@ -996,6 +992,9 @@ void CEventQueue::ServiceEvents( void )
 		// restart the list (to catch any new items have probably been added to the queue)
 		pe = m_Events.m_pNext;	
 	}
+
+	if (!engine->IsPaused())
+		m_iRealTickCount++;
```

Comment:
Should this just count paused ticks and subtract from gpGlobals->tickcount? Since tickcount won't update during pause.